test/models/role_test.rb

```diff
@@ -213,7 +213,6 @@ def test_allow_to_modify_roles_when_bypass_locking
       end
 
       test "Admin user should query Role model with no restrictions" do
-        Role.expects(:where).with('0 = 0')
```

Comment:
I guess we need a new expectation here: otherwise the test just doesn't make sense…

Reply:
done?

Reply:
Works for me